src/atomate2/vasp/flows/phonons.py

```diff
@@ -0,0 +1,306 @@
+        keyword arguments paseed to :oj: enerate_phonon_displacements
+    run_phonon_displacements_kwargs : dict
+        Keyword arguments passed to :obj:`run_phonon_displacements__kwargs`.
+    born_maker: .BaseVaspMaker or None
+        Maker to compute the BORN charges.
+    born_manual: Matrix3D
+        Instead of recomputing born charges and epsilon,
+         these values can also be provided manually. born_maker has to
+        be None
+        if full_born==False, it will expect Phonopy convention for symmetrically
+        inequivalent atoms
+        if fullborn==True, it will expect VASP convention with information for
+        every atom in unit cell. Please be careful when converting
+        structures within in this workflow
+    full_born: bool
+        reduced born file (only symmerically inequivalent atoms)
+         or full information on born
+    epsilon_static_manual: Matrix3D
+        Instead of recomputing born charges and epsilon,
+         these values can also be provided manually. born_maker has to
+        be None
+    phonon_displacement_maker : .BaseVaspMaker or None
+        Maker used to compute the forces for a supercell.
+    generate_frequencies_eigenvectors_kwargs : dict
+        Keyword arguments passed to :obj:`generate_frequencies_eigenvectors`.
+    kpath_scheme: str
+        scheme to generate kpoints. Please be aware t
+        hat you can only use seekpath with any kind of cell
+    code: str
+        determines the dft code. currently only vasp is implemented
+    """
+
+    # TODO: add some unit conversion factors
+    #  to easily use other codes to compute phonons?
+    name: str = "phonon"
+    sym_reduce: bool = True
+    symprec: float = SETTINGS.SYMPREC
+    displacement: float = 0.01
+    min_length: float | None = 20.0
+    supercell_matrix: Matrix3D | None = None
+    use_primitive_standard_structure: bool = False
+    use_conventional_standard_structure: bool = False
+    bulk_relax_maker: BaseVaspMaker | None = field(
+        default_factory=lambda: DoubleRelaxMaker.from_relax_maker(TightRelaxMaker())
+    )
+    static_energy_maker: BaseVaspMaker | None = field(default_factory=StaticMaker)
+    generate_phonon_displacements_kwargs: dict = field(default_factory=dict)
+    run_phonon_displacements_kwargs: dict = field(default_factory=dict)
+    born_maker: BaseVaspMaker | None = field(default_factory=DielectricMaker)
+    born_manual: List[Matrix3D] | None = None
+    full_born: bool = True
+    epsilon_static_manual: Matrix3D | None = None
+    phonon_displacement_maker: BaseVaspMaker = field(
+        default_factory=PhononDisplacementMaker
+    )
+    create_thermal_displacements: bool = True
+    generate_frequencies_eigenvectors_kwargs: dict = field(default_factory=dict)
+    kpath_scheme: str = "seekpath"
+    code: str = "vasp"
+
+    def make(
+        self,
+        structure: Structure,
+        prev_vasp_dir: str | Path | None = None,
```

Comment:
Instead of using `born_manual`, `epsilon_static_manual`, and `supercell_matrix` as dataclass fields, can you add `born`, `epsilon_static` and `supercell_matrix` to the make function arguments instead. And only calculate born etc if these are set to `None`.